based on commit 5a647459c82250dfd382b3bda7ec909c0e581b23
Author: Andreas Schwab <schwab@redhat.com>
Date:   Mon Sep 6 14:55:59 2010 +0200

    Don't mix pattern rules with normal rules
    (cherry picked from commit 32cf40699346d37fabfa887bbd95e95004799ae1)

diff --git a/manual/Makefile b/manual/Makefile
index c5866eb9de..b1f5fa73e5 100644
--- a/manual/Makefile
+++ b/manual/Makefile
@@ -232,7 +232,10 @@ ifdef objpfx
 .PHONY: stubs
 stubs: $(objpfx)stubs
 endif
-$(objpfx)stubs ../po/manual.pot $(objpfx)stamp%:
+$(objpfx)stubs ../po/manual.pot:
+	$(make-target-directory)
+	touch $@
+$(objpfx)stamp%:
 	$(make-target-directory)
 	touch $@

